model/project.go

```diff
@@ -37,7 +37,8 @@ const (
 )
 
 type GetProjectTasksOpts struct {
-	Limit        int    `json:"num_versions"`
+	NumVersions  int    `json:"num_versions"`
```

Comment:
We can't change limit's json; this will break existing usages.

Reply:
Reverted and removed NumVersions per the other comment